### PR TITLE
Built console-friendly csv to sqlite3 converter

### PR DESCRIPTION
**Pull request by inesusvet**
This approach helps save the open data to sqlite3 database for further aggregation and processing

Works well with python 3.5.2
Can read csv data stream from file or standard input
Call it with `-h` argument to see help page

@davojta

### CODE REVIEW

File: scripts/import_sqlite.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+def filter_row(data, known, wanted):
+    return OrderedDict({
+        key: data[key]
+        for key in known
+        if key in wanted
+    })
+
+
+def bulk_insert_stmt(table_name, keys, rows):
```
Comment (inesusvet) [inline scripts/import_sqlite.py]: This approach uses O(n) memory footprint 'cause all the rows gonna be processed as one insert-statement.
The query string could be quite long!

**Review by inesusvet — COMMENTED** [review]
Examples of calling script from repo's root
> $ python3 scripts/import_sqlite.py db.lite forests -c city,district,forest_area,district_area -n forest_area,district_area data/sources/mensk-forests.csv -v

> $ python3 scripts/import_sqlite.py db.lite amenities -c city,district,amenity,count -n count - < data/sources/mensk-amenities.csv

File: scripts/import_sqlite.py (hunk 1 above)
Comment (inesusvet) [reply scripts/import_sqlite.py]: Reworked this with `cursor.executemany()` and generator
see [its documentation](https://docs.python.org/3.5/library/sqlite3.html#sqlite3.Cursor.executemany)

**Comment by inesusvet** [comment]
Example of sql-query from console
`sqlite3 db.lite -header -csv 'SELECT district, forest_area/district_area FROM forests'`
> district,forest_area/district_area
"Заводской район",0.298946462185967
"Октябрьский район",0.0984059340460058
"Московский район",0.00826319759687542
"Центральный район",0.127752800812382
"Советский район",0.0702537839471812
"Партизанский район",0.567950688095135
"Ленинский район",0.0549170704028064
"Фрунзенский район",0.0353099822376492
"Первомайский район",0.183994957862597

**Review by davojta — APPROVED** [review]

**Pull request merged**